src/test/scala/uk/gov/hmrc/test/ui/pages/base/CacheHelper.scala

```diff
@@ -26,7 +26,10 @@ trait CacheHelper {
 
   def clear(): Unit = cache.clear()
 
-  def clear(sectionId: Int): Cache = cache --= allSectionDetails(sectionId).keys
+//  def clear(sectionId: Int): Cache = cache --= allSectionDetails(sectionId).keys
```

Comment:
This should be removed. No point in leaving it commented.

Reply:
done